lib/ProMotion/menu/transition.rb

```diff
@@ -0,0 +1,45 @@
+module ProMotion; module Menu
+  module Transition
+
+     VISUAL_STATES = {
+      slide_and_scale:
+        {
+          block: MMDrawerVisualState.slideAndScaleVisualStateBlock,
```

Comment:
Any particular reason for the nested hashes here? It seems like we should be able to do something like:

```ruby
slide_and_scale: MMDrawerVisualState.slideAndScaleVisualStateBlock,
slide: MMDrawerVisualState.slideVisualStateBlock,
# ...
```